pkg/scannerv4/mappers/mappers_test.go

```diff
@@ -1130,7 +1130,7 @@ func Test_toProtoV4VulnerabilitiesMapWithEPSS(t *testing.T) {
 			enableEPSS := "true"
 			t.Setenv(features.EPSSScore.EnvVar(), enableEPSS)
 			got, err := toProtoV4VulnerabilitiesMap(ctx, tt.ccVulnerabilities, tt.nvdVulns, tt.epssItems, nil)
-			assert.NoError(t, err)
+			require.NoError(t, err)
```

Comment:
I think we usually keep this as an `assert`, no?

Reply:
Yeah, I confused with the one above